hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -487,6 +487,8 @@ private int getBlockCount(List<ExtendedBlocklet> blocklets) {
     // First prune using default datamap on driver side.
     TableDataMap defaultDataMap = DataMapStoreManager.getInstance().getDefaultDataMap(carbonTable);
     List<ExtendedBlocklet> prunedBlocklets = null;
+    // This is to log the event, so user will know what is happening by seeing logs.
+    LOG.info("Started block pruning ...");
```

Comment:
Instead of adding these logs, I think we'd better add the time consumed for pruning in statistics.

Reply:
log will anyways have timestap, we can subtract stop and start time. I have another non-default datamap PR. I will check about this.